packages/core/src/Asset/types/NftMetadata.ts

```diff
@@ -7,6 +7,9 @@ export const Uri = (uri: string) => {
   if (/^[a-z]+:\/\/.+/.test(uri)) {
     return uri as unknown as Uri;
   }
+  if (uri.startsWith('data:')) {
```

Comment:
Need to update the `InvalidStringError` message to also allow `data:`

Reply:
79a11d500edcedf286d2384ef9ed858047b97707